packages/@webex/plugin-meetings/test/unit/spec/rtcMetrics/index.ts

```diff
@@ -70,4 +73,8 @@ describe('RtcMetrics', () => {
 
     assert.callCount(webex.request, 1);
   });
+
+  it('should anonymize IP addresses', () => {
```

Comment:
We'd want one more test that also ensures that addMetrics is calling anonymizeIp so that doesn't get removed accidentally.